it/uk/gov/hmrc/helpers/controllers/SubscriptionControllerSpec.scala

```diff
@@ -569,57 +596,48 @@ class SubscriptionControllerSpec extends IntegrationSpec with AuthHelpers with M
         stubbedPost(s"""$baseURI$subscriptionURI$safeId""", BAD_REQUEST, failureResponse.toString)
         stubbedPut(s"/enrolment-store-proxy/enrolment-store/enrolments/$enrolmentKey", NO_CONTENT)
 
-        val resp: WSResponse = await(client(controllerUrl).post(jsonOrgPostData))
+        val resp: WSResponse = await(client(controllerUrl).post(awrsSubscriptionDataOrg))
         resp.status mustBe 202
       }
     }
 
     "AWRS feature flag is true for an individual" in {
 
-      val jsResultString =
-        """
-          |{
-          |  "sapNumber": "1234567890",
-          |  "safeId": "XE0001234567890",
-          |  "regimeIdentifiers": [
-          |    {
-          |      "regimeName": "AWRS",
-          |      "regimeRefNumber": "XAAW00000123456"
-          |    }
-          |  ],
-          |  "isEditable": true,
-          |  "isAnAgent": false,
-          |  "isAnIndividual": true,
-          |  "individual": {
-          |        "firstName": "Example",
-          |        "lastName": "Exampleson",
-          |        "dateOfBirth": "1980-12-25"
-          |        },
-          |  "addressDetails": {
-          |    "addressLine1": "100 SomeStreet",
-          |    "addressLine2": "Wokingham",
-          |    "addressLine3": "Surrey",
-          |    "addressLine4": "London",
-          |    "postalCode": "AA1 1AA",
-          |    "countryCode": "GB"
-          |  },
-          |  "contactDetails": {}
-          |}
-        """.stripMargin
-
       val successResponse: JsValue = Json.parse(
         s"""{"processingDate":"2015-12-17T09:30:47Z","etmpFormBundleNumber":"123456789012345","awrsRegistrationNumber": "XAAW00000123456"}"""
       )
 
       FeatureSwitch.enable(AWRSFeatureSwitches.regimeCheck())
       stubShowAndRedirectExternalCalls("Individual")
 
-      stubbedGet(s"""$regimeURI""", OK, jsResultString)
+      stubbedGet(s"""$regimeURI""", OK, etmpSubscriptionResult)
       stubbedPost(s"""$baseURI$subscriptionURI$safeId""", BAD_REQUEST, successResponse.toString)
       stubbedPut(s"/enrolment-store-proxy/enrolment-store/enrolments/$enrolmentKey", NO_CONTENT)
 
-      val resp: WSResponse = await(client(controllerUrl).post(jsonIndividualPostData))
+      val resp: WSResponse = await(client(controllerUrl).post(awrsSubscriptionDataInd))
       resp.status mustBe 202
     }
   }
+
+  "return 400" when {
+    "AWRS Feature flag is true and organisation details match but enrolment cannot be created" in {
```

Comment:
Is it overkill to do this with Individual as well - only that we seem to have covered tests above for Org and Individual